data/CA/incentives.json

```diff
@@ -0,0 +1,2817 @@
+    "amount": {
+      "type": "dollar_amount",
+      "number": 250,
+      "maximum": 250
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Get up to $250 cash back on your induction range or cooktop upgrade.  Must replace existing gas range."
+    }
+  },
+  {
+    "id": "CA-120",
+    "authority_type": "other",
+    "authority": "ca-bay-area-regional-energy-network-bay-ren",
+    "eligible_geo_group": "ca-group-bay-area-regional-energy-network-bay-ren",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "ca_bayAreaRegionalEnergyNetwork(BayREN)_rebates:SingleFamilyHomeowners",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 0.75,
+      "unit": "square_foot",
+      "maximum": 1000
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$0.75 / sqft rebate (not to exceed $1,000 per home) for attic insulation."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "CA-121",
+    "authority_type": "other",
+    "authority": "ca-bay-area-regional-energy-network-bay-ren",
+    "eligible_geo_group": "ca-group-bay-area-regional-energy-network-bay-ren",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "ca_bayAreaRegionalEnergyNetwork(BayREN)_rebates:SingleFamilyHomeowners",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 0.7,
+      "unit": "square_foot",
+      "maximum": 1000
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$0.70 / sqft rebate (not to exceed $1,000 per home) for wall insulation."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "CA-122",
+    "authority_type": "other",
+    "authority": "ca-bay-area-regional-energy-network-bay-ren",
+    "eligible_geo_group": "ca-group-bay-area-regional-energy-network-bay-ren",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "ca_bayAreaRegionalEnergyNetwork(BayREN)_rebates:SingleFamilyHomeowners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 150
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Get $150 cash back on air sealing when you work with a BayREN Participating Contractor."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "CA-123",
+    "authority_type": "other",
+    "authority": "ca-bay-area-regional-energy-network-bay-ren",
+    "eligible_geo_group": "ca-group-bay-area-regional-energy-network-bay-ren",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "other",
+    "program": "ca_bayAreaRegionalEnergyNetwork(BayREN)_rebates:SingleFamilyHomeowners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 500
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$500 rebate for duct replacement."
```

Comment:
Ooh, this is a new one. Good to leave it as "other" for now, but we'll probably want a new `item` value for this.